packages/components/src/Columns/Columns.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Children } from 'react';
+import { useStyles } from '@marigold/system';
+import { Box } from '../Box';
+import flattenChildren from 'react-keyed-flatten-children';
+
+type ColumnsProps = {
+  className?: string;
+  space?: number;
+  horizontalAlign?: 'left' | 'right' | 'center';
+  verticalAlign?: 'top' | 'bottom' | 'center';
+  title?: string; // Should only be used for testing.
+};
+
+export const Columns: React.FC<ColumnsProps> = ({
+  space = 0,
+  horizontalAlign = 'left',
+  verticalAlign = 'top',
+  className,
+  children,
+  ...props
+}) => {
+  let columnItems = flattenChildren(children);
+  let childClassNames = useStyles({ p: `${space / 2}px` });
+
+  // horizontal Alignment
+  let justify = 'flex-start';
```

Comment:
We use const not let

Reply:
Ok but then the assignee doesn't work. Should I use state vars?

Reply:
or just var?

Reply:
Hu? What's the error?

Reply:
Error in the two lines below: Cannot assign to 'justify' because it is a constant.
In the past we used var in such cases

Reply:
oh yeah sorry, I didn't caught that. I was confused by line 22-23 because you used `let` there too.